app/components/lazy-minted/LazyMinted.tsx

```
@@ -0,0 +1,99 @@
+                  {nft.supply.toString()}
+                </td>
+                <td className="border border-slate-700 px-4 py-1">{'->'}</td>
+              </tr>
+            ))}
+          </tbody>
+        </table>
+      )}
+
+      {isModalOpen && (
+        <ConfigureMintedAsset
+          nft={nft!!}
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)